### ISSUE TITLE
Troubles with assertions on bodyjson

### ISSUE THREAD

**Issue by driccio** [ovh/venom#262]
Hello,

I am trying using venom on my project, and I have some troubles with assertions in a http step.

Assuming my result is a json like that :+1: 
`
{
   "myProperty": [
      {}
    ]
}`

Here some assertions I tried :
```
assertions:
  - result.bodyjson.myproperty.__type__ ShouldEqual Array       #A1
  - result.bodyjson.myproperty.__len__ ShouldEqual 1            #A2
  - result.bodyjson.myproperty ShouldNotBeEmpty                 #A3
  - result.bodyjson.myproperty ShouldHaveLength 1               #A4
```

A1 and A2 -> it works. But I found these examples in your tests. Is there a doc anywhere ?
A3 and A4 -> it fails. I don't understand why reading the error message : Could not access "result.bodyjson.myproperty" in assertion "result.bodyjson.myproperty ShouldHaveLength 1". The message for A3 is similar.

I would be pleased to understand what is wrong in my sample. Thanks.

**Comment by yesnault** [comment]
if you want to check an array, __type__ and __len__ are the good variables. ShouldNotBeEmpty and ShouldHaveLength don't work with an array.
I keep this issue opened until doc or something else is fixed about that.

**Comment by fsamin** [comment]
should be fixed in 47441c3
please reopen the issue if you are still have issues
